### PR TITLE
Adding an image label for VS2015.3

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @mmitche, @jasonmalinowski, @jaredpar, @dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by tannergooding** [comment]
If we are prepared to move everyone over to update 3 right now, we can instead change the 'latest-or-auto' label.

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -162,6 +162,8 @@ class Utilities {
                                 // to the auto images, at which point we will move back to
                                 // the generic unversioned label except for special cases.
                                 'latest-or-auto':'auto-win2012-20160325',
+								// VS2015.3 image:
```
Comment (jasonmalinowski) [inline jobs/generation/Utilities.groovy]: Indenting seems off.

Comment (jaredpar) [inline jobs/generation/Utilities.groovy]: Did he use ... TABS????

Comment (tannergooding) [inline jobs/generation/Utilities.groovy]: It would help if this repo had a proper `.editorconfig` file 😄

**Pull request merged**